tools/example_scripts/deletemincsqlwrapper.pl

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use Getopt::Tabular;
+no warnings 'once';
+use Data::Dumper;
+use File::Basename;
+use File::Copy;
+use Term::ANSIColor qw(:constants);
+use NeuroDB::DBI;
+
+my $profile = undef;
+my $insertminc;
+
+my @opt_table           = (
+    [ "Basic options", "section" ],
+    [
+        "-profile", "string", 1, \$profile,
+        "name of config file in ../dicom-archive/.loris_mri"
+    ],
+    [   
+         "-insertminc", "boolean", 0, \$insertminc, "Re-insert the deleted minc"
+    ]
+);
+
+my $Help = <<HELP;
+******************************************************************************
+Wrapper to minc_deletion.pl for bulk deletion based on a SQL customisable Query
+******************************************************************************
+
+This script is a wrapper for deleting multiple mincs at a time and optionally 
+re-inserting them.
+
+It will pause before for confirmation before deleting.
+
+HELP
+
+my $Usage = <<USAGE;
+usage: tools/deletemincsqlwrapper.pl -profile prod
+       $0 -help to list options
+USAGE
+&Getopt::Tabular::SetHelp( $Help, $Usage );
+&Getopt::Tabular::GetOptions( \@opt_table, \@ARGV ) || exit 1;
+
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/" . $profile}
+my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+
+
+
+# Only the f.SeriesUID is really needed for minc_deletion, other fields are for information only
+# If you plan to re-insert, you'll also need ArchiveLocation
+my $queryF = <<SQL;
+  select distinct f.fileid, f.SeriesUID, f.SessionID, f.file, t.ArchiveLocation, from_unixtime(f.InsertTime), p.Value, q.QCStatus, c.Alias, m.Scan_type
+  from files as f 
+  left join parameter_file as p using (FileID)
+  left join parameter_type as pt using (ParameterTypeID)
+  left join files_qcstatus as q using (FileID)
+  left join session as s on (f.SessionID=s.ID)
+  left join psc as c on (c.CenterID=s.CenterID)
+  left join mri_scan_type as m on (m.ID=f.AcquisitionProtocolID)
+  left join tarchive AS t on f.TarchiveSource=t.TarchiveID
+  where pt.Name = 'acquisition:slice_thickness'
+  and p.Value like '%4.%'
+  and (m.Scan_type like '%t1%' or m.Scan_type like '%t2%')
+  order by from_unixtime(f.InsertTime)
```

Comment:
Would be better if the SQL query were properly formatted using capitalized letter for mysql keywords.

SELECT DISTINCT blablabla
FROM files AS f
LEFT JOIN blabla AS bla
etc...